testing-unit-common/build.gradle

```diff
@@ -38,11 +38,9 @@ android {
 
 dependencies {
     api project(':core')
-    api libs.androidx.test.rules
 
-    implementation libs.androidx.lifecycle.java8
     implementation libs.androidx.lifecycle.runtime
-    implementation libs.compose.ui.ui
     implementation libs.junit
```

Comment:
Can we remove the junit dependancy?

Reply:
No, we are using it inside `ParentNodeTestHelper`

Reply:
Hmm. Then really this is a junit common module, not a unit test common module.

I.e. someone who was using spek they might not want to use this module.

Reply:
Fair point. I just renamed the module :)